scripts/compile_flattened.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -ex
+
+PRAGMA="// SPDX-License-Identifier:MIT\npragma solidity ^0.6.10;\npragma experimental ABIEncoderV2;"
+CONTRACTS=("RelayHub" "StakeManager" "Penalizer" "Forwarder" "TestPaymasterEverythingAccepted")
+
+# ok, I have no idea how to do it right; if it's working it ain't stupid though
+PATHS=("./" "./" "./" "./forwarder/" "./test/")
```

Comment:
I agree. you proved your point - you're not good at writing shell scripts....

but seriously, I don't think we should use this method as main compile-for-deployment, so it doesn't really matter.

Reply:
FWIW, I'd give the paths tot he contract names themselves, instead of having two arrays scanned by index, and merge them...
```
for c in RelayHub StakeManager forwarder/Forwarder test/TestPaymaster; do echo $c.sol; done
```